strawberryfields/decompositions.py

```diff
@@ -15,6 +15,7 @@
 
 from itertools import groupby
 
+import math as mt
```

Comment:
Since only one function is being used from the math library, perhaps better to be explicit here and have `from math import fmod`

Reply:
What @josh146  says sounds like a better idea than importing all of math. Why do you prefer the fmod from python over the fmod from numpy?